compiler-plugin/src/main/java/io/ballerina/stdlib/http/compiler/HttpDiagnosticCodes.java

```diff
@@ -79,6 +79,7 @@
     HTTP_130("HTTP_130", "invalid usage of cache annotation with return type : '%s'. " +
             "Cache annotation only supports return types of anydata and SuccessStatusCodeResponse", ERROR),
     HTTP_131("HTTP_131", "invalid usage of payload annotation with return type : '%s'", ERROR),
+    HTTP_132("HTTP_132", "%s should have a resource method", ERROR),
```

Comment:
```suggestion
    HTTP_132("HTTP_132", "%s must have a resource method", ERROR),
```

Reply:
Fixed in [`e22bc07`](https://github.com/ballerina-platform/module-ballerina-http/pull/893/commits/e22bc07f24c310488bed719cbba29e197f38b36d)